elastic/kibana
# [Infra] Add observability feature id to host request

## Summary

Fixes the Hosts view for serverless by enabling the `observability` feature id for the `/api/metrics/infra/{assetType}` request.

Closes #191078 

## How to test

- Go to Infrastructure -> Hosts
- Go to Alerts & Rules, Infrastructure Rules and create a new Inventory Rule, something like alert on cpu usage above certain %.
- Allow the rule/alert to run
- Go back to Infrastructure -> Hosts view
- Hosts table should have a column with alert icons now visible for all hosts with alerts active on them.

This should be tested against serverless + stateful versions of the Observability product.

https://github.com/user-attachments/assets/8e0b36e0-30e6-4176-8190-592b3aaf2732
